### PR TITLE
firewall-rule Contrail API

### PR DESCRIPTION
**Pull request by jablonskim**

### CODE REVIEW

File: Test/Utils/ContrailAPI/Models/FirewallService.ps1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Those are just informative to show dependencies
+#include "Protocol.ps1"
+#include "PortRange.ps1"
+
+class FirewallService {
+    [Protocol] $Protocol = [Protocol]::any
```
Comment (jacekchalupka) [inline Test/Utils/ContrailAPI/Models/FirewallService.ps1]: Why default value when there is only one constructor that sets $Protocol?

**Comment by jablonskim** [comment]
recheck no bug

**Comment by jablonskim** [comment]
recheck no bug

**Comment by jablonskim** [comment]
recheck no bug

**Pull request merged**